src/utils/connectionStringParser.ts

```diff
@@ -0,0 +1,47 @@
+import { ConnectionString, ConnectionStringKey } from '../Declarations/Contracts';
+
+import * as Constants from '../Declarations/Constants';
+
+export class ConnectionStringParser {
+  private static readonly FIELDS_SEPARATOR = ';';
+
+  private static readonly FIELD_KEY_VALUE_SEPARATOR = '=';
+
+  public static parse(connectionString?: string): ConnectionString {
+    if (!connectionString) {
+      return {};
+    }
+
+    const kvPairs = connectionString.split(ConnectionStringParser.FIELDS_SEPARATOR);
+
+    const result: ConnectionString = kvPairs.reduce((fields: ConnectionString, kv: string) => {
+      const kvParts = kv.split(ConnectionStringParser.FIELD_KEY_VALUE_SEPARATOR);
+
+      if (kvParts.length === 2) {
+        // only save fields with valid formats
+        const key = kvParts[0].toLowerCase() as ConnectionStringKey;
+        const value = kvParts[1];
+        return { ...fields, [key]: value };
+      }
+      return fields;
+    }, {});
+
+    if (Object.keys(result).length > 0) {
+      // this is a valid connection string, so parse the results
+
```

Comment:
Check for `Authorization=ikey`?

Reply:
afaik it defaults to `ikey` when not specified. This exporter won't handle any other values of it for now

Reply:
Yes but a user could in theory pass in a value that is NOT `ikey` -> `Authorization=invalid` which is technically an invalid connection string.

Reply:
makes sense. Added a warning here